tests/system-fetcher/tests/handler_test.go

```diff
@@ -47,6 +47,8 @@ const (
 		"systemNumber": "%d",
```

Comment:
How much effort would it be to have the e2e test which ensures that the many to many table is leveraged when one and the same app is bound to multiple tenants?